README.md

```diff
@@ -0,0 +1,19 @@
+## Babel Sandbox
+
+### Quick start:
+
+- Install dependencies `yarn install` or `npm install`
+- Start front-end `yarn start` or `npm start`
+    - Output: Web page using `localhost:3000` should open
+
+To start the server, visit [babel-sandbox-server](https://github.com/MLH-Fellowship/babel-sandbox-server) and follow the steps in [Quick Start](https://github.com/MLH-Fellowship/babel-sandbox-server#quick-start).
+
+### Directory structure:
+```
+/public 	
+/src 
+	/components 				
+	/reducers
+	/helpers
```

Comment:
I've never been a big fan of having a `helpers` folder, since it encourages us to dump everything in there. This article was written for Go but can be applied to JS [(Article)](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common). However, I think the rest of the team should weigh in on this.

Reply:
That's fair, I don't expect us to have a lot of helpers - was more of a way to clean up the parent folder. 

Thoughts @ianjennings @mohammedsahl @kirbygit ?

Reply:
As it stands I'm not using the helpers folder in the server repo and I don't plan on doing so
Also, you could also use the awesome [0.2.1 readme directory generator](https://project-structure-readme.netlify.app/) 😉 😉

Reply:
Helpers are cool. I vote to make directories when you need them. Never make an empty directory unless you're gonna write a file there.